infrastructure/docker/traffic_ops/run.sh

```diff
@@ -36,216 +38,255 @@
 # TODO:  Unused -- should be removed?  TRAFFIC_VAULT_PASS
 
 # Check that env vars are set
-envvars=( MYSQL_IP MYSQL_PORT MYSQL_ROOT_PASS MYSQL_TRAFFIC_OPS_PASS ADMIN_USER ADMIN_PASS CERT_COUNTRY CERT_STATE CERT_CITY CERT_COMPANY DOMAIN)
+envvars=( DB_SERVER DB_PORT DB_ROOT_PASS DB_USER DB_USER_PASS ADMIN_USER ADMIN_PASS CERT_COUNTRY CERT_STATE CERT_CITY CERT_COMPANY DOMAIN)
 for v in $envvars
 do
 	if [[ -z $$v ]]; then echo "$v is unset"; exit 1; fi
 done
 
 start() {
-		service traffic_ops start
-		exec tail -f /var/log/traffic_ops/traffic_ops.log
+	service traffic_ops start
+	exec tail -f /var/log/traffic_ops/traffic_ops.log
 }
 
 init() {
-		mysql -h $MYSQL_IP -P $MYSQL_PORT -u root -p$MYSQL_ROOT_PASS -e "GRANT ALL ON * . * TO 'traffic_ops'@'localhost' IDENTIFIED BY '$MYSQL_TRAFFIC_OPS_PASS';"
-
-		printf '\
-#!/usr/bin/expect -f\n \
-set force_conservative 0  ;# set to 1 to force conservative mode even if\n\
```

Comment:
glad we can get rid of the expect script..  this is much better...

Reply:
+1